packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1799,8 +1799,7 @@ async function handler(
 
     evt.appsStatus = handleAppsStatus(results, booking, reqAppsStatus);
 
-    // If there is an integration error, we don't send successful rescheduling email, instead broken integration email should be sent that are handled by either CalendarManager or videoClient
-    if (noEmail !== true && isConfirmedByDefault && !isThereAnIntegrationError) {
```

Comment:
I did not receive any broken integration mail when google calendar integration fails

Reply:
The event was not updated as credentials don't work and i got successfull reschedule email

Reply:
Yeah, that isn't fixed in this PR. That's a separate task pending from a long time now 
![image](https://github.com/calcom/cal.com/assets/1780212/060565ff-4c9b-443b-a969-15183e40a1fd)